library/src/animationManager.cxx

```diff
@@ -237,6 +237,24 @@ void animationManager::Tick()
     {
       this->Window.render();
     }
+
+    // Check if camera orbit option is enabled
+    if (this->Options.scene.camera.orbitTime > 0)
```

Comment:
I dont think camera orbit should be handled in animation, but it should be handled in the eventLoop directly.

Reply:
I even think the camera orbit should be handled in the application event loop, not in the lib